tests/test_explorer.py

```diff
@@ -15,6 +15,7 @@
 class FileFinderThreadTest(unittest.TestCase):
     """Unit tests for _FileFinderThread class."""
 
+    # pylint: disable=duplicate-code
```

Comment:
Really?

I added the comments in both `test_explorer.py` and `test_builder.py` because the `setUp()` methods are the same and caused the Pylint warning.

Reply:
Oh, really? I do not have any pylint disable comment in my branch for [PR#89](https://github.com/snu-quiqcl/iquip/pull/89), but it did not give any pylint warning.